marketing/views.py

```diff
@@ -0,0 +1,14 @@
+from django.views.generic import TemplateView
+
+from core.mixins import GetCMSPageMixin
+
+
+class CampaignPageView(
+    GetCMSPageMixin,
+    TemplateView
```

Comment:
these could all be one line - would be easier to read as `class CampaignPageView(GetCMSPageMixin, TemplateView)`

Reply:
I suspect this needs to be feature flagged